services/QuillLMS/client/app/bundles/Teacher/components/classrooms/reauthorize_clever_modal.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react'
+
+interface ReauthorizeCleverModalProps {
+  cleverLink: string;
+  close: () => void;
+}
+
+export default class ReauthorizeCleverModal extends React.Component<ReauthorizeCleverModalProps, {}> {
+  constructor(props) {
+    super(props)
+    this.handleReauthorizeClick = this.handleReauthorizeClick.bind(this)
+  }
```

Comment:
Shouldn't use bind here - and in fact you don't need a constructor at all.

Reply:
`constructor` always gets called under the hood, so all you're doing here once you remove the unnecessary bind is defining it as calling `super`, which is the same as not including it.